### PR TITLE
fix: fix recent feedback items

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [ ] Not that difficult
- [ ] Approved feature branch merge to master

### Description
Plz, read order by commits

### CODE REVIEW

File: apps/web/src/common/modules/navigations/top-bar/modules/top-bar-search/modules/top-bar-search-dropdown/TopBarSearchDropdown.vue [hunk 1]
```diff
@@ -96,7 +96,7 @@ const handleUpdateActiveTab = (tab: SearchTab) => {
 };
 
 const handleUpdateContentsSize = (height: number) => {
-    state.contentsHeight = height;
+    if (state.contentsHeight !== height) state.contentsHeight = height;
 };
```
Comment (yuda110) [inline apps/web/src/common/modules/navigations/top-bar/modules/top-bar-search/modules/top-bar-search-dropdown/TopBarSearchDropdown.vue:100]: suggestion: How about applying `debounce` here?

File: apps/web/src/lib/helper/router-recent-helper.ts [hunk 2]
```diff
@@ -25,7 +25,9 @@ export const getRecentConfig = (to: Route): RecentConfig | undefined => {
     if (to.name === DASHBOARDS_ROUTE.DETAIL._NAME) {
         const dashboardId = to?.params?.dashboardId;
         if (!dashboardId) return undefined;
-        return { itemType: RECENT_TYPE.DASHBOARD, workspaceId, itemId: dashboardId };
+        const isPublicDashboard = dashboardId.slice(0, 6) === 'public';
```
Comment (yuda110) [inline apps/web/src/lib/helper/router-recent-helper.ts:28]: suggestion: How about to use `dashboardId.startsWith('public')`?

File: apps/web/src/services/asset-inventory/components/ServiceAccountDeleteModal.vue [hunk 3]
```diff
@@ -50,10 +52,14 @@ const state = reactive({
 /* Api */
 const deleteServiceAccount = async () => {
     try {
-        if (state.isGeneralAccount && ('service_account_id' in props.serviceAccountData)) {
+        if (state.isGeneralAccount && props.serviceAccountData && ('service_account_id' in props.serviceAccountData)) {
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/components/ServiceAccountDeleteModal.vue:55]: suggestion: How about
```typescript
'service_account_id' in props.serviceAccountData

// tobe
props.serviceAccountData?.service_account_id
```

**Review by yuda110 — APPROVED** [review]
LGTM!! I left some comments!!

File: apps/web/src/services/asset-inventory/components/ServiceAccountDeleteModal.vue (hunk 3 above)
Comment (sulmoJ) [reply apps/web/src/services/asset-inventory/components/ServiceAccountDeleteModal.vue:55]: This code had to be written as above for TypeScript errors.

**Pull request merged**